src/app/legacy/components/Promo/index.test.jsx

```diff
@@ -77,3 +77,120 @@ describe('Promo component - Image', () => {
+        );
+        expect(getByText('1 minuto')).toBeInTheDocument();
+      });
+      it('should render 1 minute ago in epoch format if time since last published is less than 1 minute ago', () => {
+        const zeroMinuteAgoEpoch = calcTimestampMinutesAgo(0).getTime();
+        const { getByText } = render(
+          <Fixture timestamp={zeroMinuteAgoEpoch} />,
+          {
+            service: 'mundo',
+          },
+        );
+        expect(getByText('1 minuto')).toBeInTheDocument();
+      });
+    });
+    describe('Relative time 59 minutes ago', () => {
+      it('should render 59 minutes ago or less in string format', () => {
+        const fiftyNineMinutesAgoString =
+          calcTimestampMinutesAgo(59).toISOString();
+        const { getByText } = render(
+          <Fixture timestamp={fiftyNineMinutesAgoString} />,
+          {
+            service: 'mundo',
+          },
+        );
+        expect(getByText('59 minutos')).toBeInTheDocument();
+      });
+      it('should render 59 minutes ago or less in epoch format', () => {
+        const fiftyNineMinutesAgoEpoch = calcTimestampMinutesAgo(59).getTime();
+        const { getByText } = render(
+          <Fixture timestamp={fiftyNineMinutesAgoEpoch} />,
+          {
+            service: 'mundo',
+          },
+        );
+        expect(getByText('59 minutos')).toBeInTheDocument();
+      });
+    });
+    describe('Relative time 10 hours ago', () => {
+      it('should render 10 hours ago or less in string format', () => {
+        const tenHoursAgoString = calcTimestampHoursAgo(10).toISOString();
+        const { getByText } = render(
+          <Fixture timestamp={tenHoursAgoString} />,
+          {
+            service: 'mundo',
+          },
+        );
+        expect(getByText('10 horas')).toBeInTheDocument();
+      });
+      it('should render 10 hours ago or less in epoch format', () => {
+        const tenHoursAgoEpoch = calcTimestampHoursAgo(10).getTime();
+        const { getByText } = render(<Fixture timestamp={tenHoursAgoEpoch} />, {
+          service: 'mundo',
+        });
+        expect(getByText('10 horas')).toBeInTheDocument();
+      });
+    });
+    describe('Relative time more than 11 hours ago', () => {
+      it('should render timestamp in ISO string format', () => {
+        const overElevenHoursString = calcTimestampHoursAgo(11).toISOString();
+        const { getByText } = render(
+          <Fixture timestamp={overElevenHoursString} />,
+          {
+            service: 'mundo',
+          },
+        );
+        expect(getByText('6 noviembre 2023')).toBeInTheDocument();
+      });
+      it('should render timestamp in ISO format', () => {
+        const overElevenHoursEpoch = calcTimestampHoursAgo(11).getTime();
+        const { getByText } = render(
+          <Fixture timestamp={overElevenHoursEpoch} />,
+          {
+            service: 'mundo',
+          },
+        );
+        expect(getByText('6 noviembre 2023')).toBeInTheDocument();
```

Comment:
This is going to fail tomorrow... so we will need to either check something else in the timestamp component, or do some form of mocking of Date.now() - I can help out with this.